core/block.js

```diff
@@ -1338,7 +1338,7 @@ Blockly.Block.prototype.jsonInit = function(json) {
  * @param {boolean=} opt_disableCheck Option flag to disable overwrite checks.
  */
 Blockly.Block.prototype.mixin = function(mixinObj, opt_disableCheck) {
-  if (goog.isDef(opt_disableCheck) && !goog.isBoolean(opt_disableCheck)) {
+  if ((opt_disableCheck !== undefined) && !(typeof opt_disableCheck == 'boolean')) {
```

Comment:
You can replace
```
!(typeof opt_disableCheck == 'boolean')
```
with
```
typeof opt_disableCheck != 'boolean'
```

and do the same in many other places.

Reply:
Replaced as requested